apps/fyler/src/fyler_monitor.erl

```diff
@@ -43,11 +43,12 @@ stop_monitor() ->
 init(_Args) ->
   ?D(monitor_init),
   Pid = spawn_link(fyler_system_listener, listen, []),
-  {ok, idle, #state{listener = Pid, cpu_max = ?Config(cpu_high_watermark, 90), timeout = ?Config(cpu_check_timeout, 2000)}}.
+  {ok, idle, #state{listener = Pid, cpu_max = ?Config(cpu_high_watermark, 90), timeout = ?Config(cpu_check_timeout, 10000)}}.
```

Comment:
Line 46 is too long:   {ok, idle, #state{listener = Pid, cpu_max = ?Config(cpu_high_watermark, 90), timeout = ?Config(cpu_check_timeout, 10000)}}..